rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
```

Comment:
- [ ] You are currently missing the has-many relationship between person and rentals as highlighted in the requirements below:
![Screenshot 2023-07-05 at 10 01 07 (2)](https://github.com/Ashnaali3255/OOP-School-Library/assets/64774936/5bb598f3-5d9e-4579-a145-3259ecc5fece)

It should be added to the `Person` class and be similar to this:
```ruby
class Person
 
  attr_accessor :name, :age, :rentals
  .
  .

  def initialize(age, name = 'Unknown', parent_permission: true)
    .
    .
    @rentals = []
  end

  def add_rental(book, date)
    Rental.new(date, book, self)
  end

  .
  .
end
```